src/health_misinfo_shared/fine_tuning.py

```diff
@@ -465,6 +469,38 @@ def infer_claims(video_id: str, transcript: list[dict]) -> Iterable[dict[str, An
     return all_responses
 
 
+def infer_multimodal_claims(
+    bucket_name: str, video_path: str, model_name: str = "gemini-1.5-flash-001"
+) -> Iterable[dict[str, Any]]:
+    vertexai.init(project=GCP_PROJECT_ID, location=GCP_LLM_LOCATION)
+    model = GenerativeModel(model_name=model_name)
+    response = model.generate_content(
+        [
+            Part.from_uri("gs://" + video_path, mime_type="video/mp4"),
+            MULTIMODAL_RAPHAEL_PROMPT,
+        ],
+        safety_settings=VIDEO_SAFETY_SETTINGS,
+        generation_config={
+            "candidate_count": 1,
+            "max_output_tokens": 8192,
+            "temperature": 0,
+            "top_p": 1,
+        },
+    )
+
+    if not len(response.candidates):
+        raise Exception(f"No model output: possible reason: {response.prompt_feedback}")
+
+    candidate = response.candidates[0]
+    text = candidate.text
+    if text.startswith("```"):
+        text = text.strip("`")
+    if text.startswith("json"):
+        text = text[4:]
+    print(text)
+    return json.loads(text)
```

Comment:
Feel free to ignore this completely but there's [an existing function for dealing with the common json parsing problems](https://github.com/FullFact/health-misinfo-shared/blob/main/src/health_misinfo_shared/data_parsing.py#L26). 

It doesn't do much beyond what you've already done though so if you don't want to change stuff feel free to ignore - I just noticed it while I was looking through the code and thought I'd mention

Reply:
Oh, I missed this. I will update it to use the utility function.

Reply:
There is a [fairly popular library for this purpose](https://github.com/mangiucugna/json_repair).

Reply:
Ooh, that looks really useful!

Reply:
I’ve opened a ticket for json_repair (#176), because I think it could be treated as a separate piece of work and doesn’t need to be addressed in this PR.

Reply:
Going to leave this to be resolved by #176